drivers/hmis/spec/requests/hmis/lookup_client_spec.rb

```diff
@@ -146,6 +155,9 @@
       'addresses' => [{ 'line1' => '999 Test Ave' }],
       'phoneNumbers' => [{ 'value' => '5554567891', 'use' => 'home', 'system' => 'phone' }],
       'emailAddresses' => [{ 'value' => 'email@e.mail', 'use' => 'home', 'system' => 'email' }],
+      'image' => {
+        # TODO @MARTHA ???
```

Comment:
TO DO: Fix this test. I spent some time stuck on this and wasn't able to figure out why tags were not getting saved. 

Paused at a breakpoint in the test,
```
0> GrdaWarehouse::ClientFile.count 
=> 1

0> GrdaWarehouse::ClientFile.first.tag_list 
=> 
# why empty?

0> photo.tag_list
=> Client Headshot
# not empty...

0> GrdaWarehouse::ClientFile.first.id 
=> 307243

0> photo.id 
=> 307243
# IDs match

0> photo.tags.count 
=> 0

0> GrdaWarehouse::ClientFile.first.tags.count
=> 0
# both zero :(
```

Reply:
not sure where your breakpoint was @martha but I put one here, and saw that the `GrdaWarehouse::ClientFile.client_photos` scope is empty https://github.com/greenriver/hmis-warehouse/blob/634e0e6685e41c7d9f328e1860c8e8779d185fdb/drivers/hmis/app/graphql/types/hmis_schema/client.rb#L243

Reply:
Oh, I think you just need to associate the file with the client you're resolving:
`let!(:photo) { create :client_file }`
should be:
`let!(:photo) { create :client_file, client: my_client }`

Reply:
Thanks for looking at it, @gigxz ! I added a test above on line 142 that illustrates the issue a bit more clearly. Line 143 passes but line 144 fails - the scope is still not returning anything even when (I think) the file is tagged correctly.

Reply:
Ah, thank you @gigxz , that's good to know. I played around with it and could, indeed, get it working in the rspec test by avoiding `tagged_with`, which looks like this: https://github.com/greenriver/hmis-warehouse/pull/4222/files#diff-cfe82ea280b947f0dbbffd0f15c9e9b3fb05d9d36794e210390b315413a1ceddR27-R35
I'd be happy to implement some common solution though if we are now making this accommodation in multiple places, but yes would like to wait for @eanders's input on how to proceed.